content/rules/anatomy.md

```diff
@@ -60,7 +60,7 @@ include_in_help: true # see help_text in help message
 * **respond** (_string_) Choose a string to match or regex pattern that this rule should trigger actions for.
   * **args** (_yaml array_) The number of items in this array determines how many args you're trying to capture after the match is found. Each string captured after the match will be mapped to a variable you describe in each element of this array.
 * **hear** (_string_) Choose a string to match or regex pattern that this rule should trigger actions for. Hear will match any occurrence of the pattern in the room regardless of whether the bot is addressed. This trigger takes no arguments.
-* **schedule** (_string_) [Cronspec](https://crontab-generator.org/) for when you want a rule to run on a schedule/ on an interval.
+* **schedule** (_string_) [Cronspec](https://crontab-generator.org/) for when you want a rule to run on a schedule.
```

Comment:
we should rip out this link and link to [this](https://godoc.org/github.com/robfig/cron#hdr-CRON_Expression_Format) instead. as it takes in 6 fields not 5.